<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>339</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Minecraft整合包导出工具</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>第一步</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>选择你的 versions 文件夹（请开启版本隔离）</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_s1">
         <property name="text">
          <string>点击此处</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>第二步</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>选择你要导出的游戏版本</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_s2">
         <property name="text">
          <string>点击此处</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>第三步</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>选择附加导出设置</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="2" column="0">
          <widget class="QCheckBox" name="cb_s3_servers">
           <property name="text">
            <string>servers.dat 服务器列表</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QCheckBox" name="cb_s3_saves">
           <property name="text">
            <string>saves 存档</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="cb_s3_config">
           <property name="text">
            <string>config 配置文件</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QCheckBox" name="cb_s3_mods">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>mods</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <property name="tristate">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QCheckBox" name="cb_s3_shaderpacks">
           <property name="text">
            <string>shaderpacks 光影包</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QCheckBox" name="cb_s3_resourcepacks">
           <property name="text">
            <string>resourcepacks 资源包</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QCheckBox" name="cb_s3_options">
           <property name="text">
            <string>options.txt 游戏设置</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cb_s3_all">
         <property name="text">
          <string>* 附带文件夹下所有内容（特殊整合包可能需要）</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="title">
       <string>第四步</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>导出！</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>339</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="action_MMET"/>
    <addaction name="action_GitHub"/>
    <addaction name="separator"/>
    <addaction name="action_2"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="action_MMET">
   <property name="text">
    <string>关于 MMET</string>
   </property>
  </action>
  <action name="action_GitHub">
   <property name="text">
    <string>GitHub</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Bug汇报</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
